.github/workflows/publish_dev_package.yml

```diff
@@ -9,6 +9,9 @@ jobs:
   build-n-publish:
     name: Build and publish Python 🐍 distributions 📦 to TestPyPI
     runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
```

Comment:
This action is to publish dev package in testpypi, we certainly don't need to publish twice, so please remove 3.7 here

Reply:
It can be built perfectly locally, but the built package does not need to be uploaded into testpypi. Besides, the publishing python environment doesn't influence the actual building process. 

So please remove 3.7 here, just leave python3.8 here.